packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -480,12 +493,36 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
             }
             return !this.runtime.disposed;
         } finally {
-            if (t !== undefined) {
-                clearTimeout(t);
+            if (timeout !== undefined) {
```

Comment:
Passing the timeout to two functions and clearing it in both. Better will be clear it in same function from where it's created and passed. Additionally, if there is an exception then timeout will not be cleared.

Reply:
Done!